src/web/module/counter/component/View.tsx

```diff
@@ -1,59 +1,80 @@
 import { clsx } from 'clsx';
-import { useEffect, useState } from 'react';
+import { Dispatch, SetStateAction, useEffect, useState } from 'react';
 
 import { Counter } from '@/api/module/counter/entity';
+import { Option } from '@/api/shared/entity';
 import { getOptionsFromSurahId } from '@/api/shared/entity/surah';
-import Card from '@/web/module/counter/component/Card';
+import { Card } from '@/web/module/counter/component/Card';
 import { useData } from '@/web/module/counter/context/DataContext';
-import Form from '@/web/shared/component/Form';
+import { Form } from '@/web/shared/component/Form';
 import { useAlert } from '@/web/shared/context/AlertContext';
 
-const View = (): JSX.Element => {
-  const classNames: Record<string, string> = {
-    juz: 'text-2xl font-medium text-custom-teal',
-    juzRuler: 'mb-2 border-custom-teal',
-  };
+interface InternalProps {
+  data: Counter[];
+  // @ts-expect-error useAlert
+  hideAlert: Context<AlertContextValues>;
+  isFormVisible: boolean;
+  setIsFormVisible: Dispatch<SetStateAction<boolean>>;
+  parentSurah: Option[] | undefined;
```

Comment:
how about?
```suggestion
  parentSurah?: Option[];
```

Reply:
I think will stick as it is for now, because using this will make the InternalProps treat that variable as optional